.github/workflows/sourcify.yml

```
@@ -0,0 +1,163 @@
+name: "sourcify"
+
+on:
+  # Run using manual triggers from GitHub UI:
+  # https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow
+  workflow_dispatch:
+    inputs:
+      chain:
+        description: "Name of the chain/SOURCIFY to run against, e.g. 'ethereum' or 'tron'."
+        type: "string"
+        required: true
+        default: "ethereum"
+      network:
+        description: "Name of the sub-network to run against, e.g. 'mainnet' or 'testnet'."
+        type: "string"
+        required: true
+        default: "mainnet"
```

WDYT of providing the chain ID directly here? given that this is a publicly documented feature?
https://chainlist.org/

```suggestion
      chainID:
        description: "ID of the chain to test. For example `1` for `ethereum/mainnet`. More info: https://chainlist.org/"
        type: "number"
        required: true
```